weave-js/src/components/RadixTooltip/Tooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+
+import * as RadixTooltip from './index';
+
+export type TooltipProps = {
+  trigger: React.ReactNode;
+  content: React.ReactNode | string;
+  /**
+   * This determines if the trigger and content components should be wrapped in
+   * RadixTooltip.Trigger and RadixTooltip.Content, respectively. Otherwise, it
+   * will just assume props passed in are properly wrapped in RadixTooltip anatomy.
+   * Having this option allows a bit more flexibilty and less prop passing.
+   *
+   * Setting it to false will tell the component to not apply the RadixTooltip.Trigger
+   * and RadixTooltip.Content around the props.
+   */
+  nowrap?: boolean;
+};
+
+/**
+ * This is a more ergonomic Tooltip built on top of our RadixTooltip. If this component does not
+ * meet your needs, please refrain from modifying this and consider using RadixTooltip directly.
+ */
+export const Tooltip = ({trigger, content, nowrap = false}: TooltipProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  if (nowrap) {
+    <RadixTooltip.Root open={isOpen} onOpenChange={setIsOpen}>
+      {trigger}
+      <RadixTooltip.Portal>{content}</RadixTooltip.Portal>
+    </RadixTooltip.Root>;
+  }
+
+  return (
+    <RadixTooltip.Root open={isOpen} onOpenChange={setIsOpen}>
+      <RadixTooltip.Trigger>{trigger}</RadixTooltip.Trigger>
```

Comment:
i see that i left a note in the base RadixTooltip Trigger comp that if the trigger is a button then the `asChild` prop should be used. i wonder if there's a way to check this automatically here and include that prop as appropriate

Reply:
Here's my dilemma, I tried replacing the alerts tooltip on the nav bar and adding `asChild` cause issues with content. It would not anchor to the trigger properly and always be at the upper left corner or screen. 

<img width="324" alt="Screenshot 2024-04-17 at 11 22 15 AM" src="https://github.com/wandb/weave/assets/103448723/2273f0ef-e693-4fec-97de-977c7d2651e2">